.github/workflows/dhis2-preview-pr.yml

```diff
@@ -24,7 +24,7 @@ jobs:
             - uses: actions/checkout@v2
             - uses: actions/setup-node@v1
               with:
-                  node-version: 12.x
+                  node-version: 14.x
```

Comment:
this change is needed to be able to use `msw` library .. The current LTS is v16 so I thought it might not be a bad idea to use a newer version anyhow, but let me know if this has other implications and I can change it back.